src/Data/MongoDBMock/doggos.js

```diff
@@ -0,0 +1,75 @@
+const doggos = [
+  {
+    dogId: 'VIII-27165',
+    dogName: 'Nosek',
+    kennelName: 'Narnia Baltica',
+    pkr: 'VIII-27165',
+    registrationNumber: '5491/XLII/11',
+    breed: 'Golden Retriever',
+    dateOfBirth: new Date(2011, 4, 22),
+    colour: 'gold',
+    chipOrTatoo: 616093900044541,
```

Comment:
typo in `chipOrTatoo`
tattoo has double `t`
so right version: `chipOrTattoo`

Reply:
changed